hapi-test/src/test/java/ca/uhn/hl7v2/model/VariesTest.java

```diff
@@ -234,6 +235,48 @@ public void testObx5WithExpectedComponentUnpexpectedSubcomponentWithinPrimitiveW
 
 	}
 
+	/**
+	 * AD = ST, ST,...
+	 */
+	@Test
+	public void testObx5WithExpectedComponentUnpexpectedSubcomponentWithinPrimitiveWithoutEscaping_ShouldFixOBX2()

```

Comment:
This test tests the pre-existing behaviour but it doesn't test the case where our new property is false ie it doesn't test the new behaviour.